src/test/java/org/example/homework14/DemoTests.java

```diff
@@ -0,0 +1,206 @@
+package org.example.homework14;
+
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class DemoTests {
+    WebDriver driver;
+    private static final By ELEMENTS_CARD = By.xpath("//div[@class='card-body']/h5[text()='Elements']");
+    private static final By WEB_TABLE_MENU_ITEM = By.id("item-3");
+    private static final By RAW_DELETE_1_BUTTON = By.id("delete-record-1");
+    private static final By RAW_DELETE_2_BUTTON = By.id("delete-record-2");
+    private static final By RAW_DELETE_3_BUTTON = By.id("delete-record-3");
+    private static final By NO_TABLE_DATA_ITEM = By.className("rt-noData");
+    private static final By CHECK_BOX_MENU_ITEM = By.id("item-1");
+    private static final By HOME_TOGGLE = By.xpath("//*[@id=\"tree-node\"]/ol/li/span/button");
+    private static final By DOWNLOADS_TOGGLE = By.xpath("//*[@id=\"tree-node\"]/ol/li/ol/li[3]/span/button");
+    private static final By EXCEL_CHECKBOX = By.xpath("//*[@id=\"tree-node\"]/ol/li/ol/li[3]/ol/li[2]/span/label/span[1]");
+    private static final By RESULT_TEXT = By.className("text-success");
+    private static final By LINKS_MENU_ITEM = By.id("item-5");
+    private static final By RADIO_BUTTON_MENU_ITEM = By.id("item-2");
+    private static final By HOME_LINK = By.id("simpleLink");
+    private static final By YES_BUTTON = By.id("yesRadio");
+    private static final By IMPRESSIVE_BUTTON = By.id("impressiveRadio");
+    private static final By ADD_DATA_BUTTON = By.id("addNewRecordButton");
+    private static final By FIRST_NAME_FIELD = By.id("firstName");
+    private static final By LAST_NAME_FIELD = By.id("lastName");
+    private static final By EMAIL_FIELD = By.id("userEmail");
+    private static final By AGE_FIELD = By.id("age");
+    private static final By SALARY_FIELD = By.id("salary");
+    private static final By DEPARTMENT_FIELD = By.id("department");
+    private static final By SUBMIT_BUTTON = By.id("submit");
+    private static final String FIRST_NAME = "TestName";
+    private static final String LAST_NAME = "Test";
+    private static final String EMAIL = "example@test.com";
+    private static final String AGE = "25";
+    private static final String SALARY = "5000";
+    private static final String DEPARTMENT = "QA";
+    private static final By NAME_RAW = By.xpath("//*[@id=\"app\"]/div/div/div/div[2]/div[2]/div[3]/div[1]/div[2]/div[4]/div/div[1]");
```

Comment:
I assume you want to get the data from each cell of the added record of the table, but let's be honest, it is absolutely impossible to get the logic of the locator.
Please simplify the logic
you can get the content of all raws displayed in the screen like this
```
    var rawList = driver.findElements(By.className("rt-tr"));
```
after it you can process the text data from the List and validate it